lib/vtls/cipher_suite.c

```diff
@@ -165,12 +165,12 @@ enum {
 
 #define CS_ZIP_IDX(a, b, c, d, e, f, g, h)    \
 {                                             \
```

Comment:
Also align the `\`s here, with those shifted below?

Reply:
I also align the below section to form a straight line.